### DISCUSSION TITLE
Different font size

### DISCUSSION THREAD

**Discussion by fbaltor** [zellij-org/zellij#1306] [Q&A] [answered]
Hey! Does zellij allows for different font size between the panes? This was a major reason for why I couldn't adapt to [tmux](https://github.com/tmux/tmux/issues/1267).

**Comment by imsnif** [comment, marked as answer]
I don't think this is possible in any tool that runs inside a terminal. The font size is controlled by the terminal itself and is always uniform.

**Reply by joseluis** [reply]
This is right.

Although theoretically and leveraging upcoming sixel support a system could be devised to render using any custom fonts into a bitmap... and manage that bitmap as an alternative terminal view... I've never seen it done before though.
